### ISSUE TITLE
IOS ItemAppearing Bug

### ISSUE THREAD

**Issue by hdingmin** [twintechs/TwinTechsFormsLib#8]
in ios  i need use ItemAppearing event to load new data, but  in your  FastCellRenderer  you  overwrite
GetCell method den't use  base.GetCell  So  ItemAppearing  event den't be execute; but i use The other way

anothor Bug  I use ObservableCollection to load my data when last listview item appear。if i sliding slowly it work  but sliding fastly app crash.

**Comment by jzeferino** [comment]
I've used FastCell along with ObservableCollection and ItemAppearing  to load more data and it worked well.

For curiosity, do you had bad list performance on iOS?
I have a Xamarin Forms application runnning in Android and iOS and i only have this performance issue in Android. On iOS the list performance is good.

**Comment by georgejecook** [comment]
Yes. I had sucky performance on everything. It depends on what your definition is but any jerkiness on iOS is unacceptable to me. 

Aside from that. I find the out the box list view creates hundreds of viewcells wasting precious system resources so I see value in it for iOS until xamarin take cell virtualisation more seriously.

**Comment by georgejecook** [comment]
Is this issue closed? If so could you summarise what you did for others please?

Personally, I don't use item appearing because I use the prepare cell method override instead.

**Comment by johandep** [comment]
I have the same problem. I need to view a couple of thousand items in a listview. I use the ItemAppearing event of the ListView to load "more" items when somebody scrolls down. When I open this page or when I do a search in this page I only have to fetch the first 30 items. This makes it a very fast user experience.
